apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatItemInfoView.kt

```diff
@@ -222,6 +289,22 @@ fun ChatItemInfoView(chatModel: ChatModel, ci: ChatItem, ciInfo: ChatItemInfo, d
     }
   }
 
+  @Composable
+  fun ForwardedFromTab(forwardedFromItem: AChatItem) {
+    // LALAL SCROLLBAR DOESN'T WORK
```

Comment:
? :)

Reply:
It's something that doesn't work yet and no time to look at it closer. Every tab on that screen has invisible scrollbar inside, so it's something to look at later